src/ensembl/package.json

```diff
@@ -54,6 +54,7 @@
     "connected-react-router": "6.5.2",
     "core-js": "3.3.6",
     "dotenv": "8.2.0",
+    "ensembl-genome-browser": "https://github.com/Ensembl/ensembl-genome-browser-assets.git#wasm/7def28c42e83db0db5eacc70ab014add",
```

Comment:
How does this work? Will this line get updated every time the wasm file is changed or does it always refer to the latest version of the file?

Reply:
Sorry, the PR didn't have proper description when I announced it. I updated the description; hopefully it will answer this question.